examples/todomvc/components/itemview.go

```diff
@@ -52,25 +45,25 @@ func (p *ItemView) onToggleCompleted(event *vecty.Event) {
 func (p *ItemView) onStartEdit(event *vecty.Event) {
 	p.editing = true
 	p.editTitle = p.Item.Title
-	p.ReconcileBody()
-	p.input.Node().Call("focus")
+	vecty.Rerender(p)
+	p.input.Node.Call("focus")
 }
 
 func (p *ItemView) onEditInput(event *vecty.Event) {
 	p.editTitle = event.Target.Get("value").String()
-	p.ReconcileBody()
+	vecty.Rerender(p)
 }
 
 func (p *ItemView) onStopEdit(event *vecty.Event) {
 	p.editing = false
-	p.ReconcileBody()
+	vecty.Rerender(p)
 	dispatcher.Dispatch(&actions.SetTitle{
 		Index: p.Index,
 		Title: p.editTitle,
 	})
 }
 
-func (p *ItemView) render() vecty.Component {
+func (p *ItemView) Render() *vecty.HTML {
```

Comment:
Component rendering is now done via implementing a public `Render` method which returns `*vecty.HTML`.

Simple, and it makes sense. Beginners do not ask the question "Why does my component render another component..?". Components can now be thought of as a way of grouping HTML elements, we no longer mix the concepts of elements and components together.